iktomi/cli/base.py

```diff
@@ -77,8 +77,6 @@ def perform_auto_complete(commands):
                     k,v = splited
                 elif len(splited) == 1:
```

Comment:
Do you have test for this case? What about empty value (`--opt=`)? Shouldn't be later handled the same way as without `=` sign?